components/NavLayout/SideBar.js

```diff
@@ -14,7 +14,13 @@ import CalendarMonthOutlinedIcon from '@mui/icons-material/CalendarMonthOutlined
 import NotificationsOutlinedIcon from '@mui/icons-material/NotificationsOutlined';
 import SearchOutlinedIcon from '@mui/icons-material/SearchOutlined';
 
+<<<<<<< HEAD
```

Comment:
This looks like a VS Code merge conflict line. Double check this, I think it may break code.

Reply:
this is not present on my code, I will tackle the other comments and repush. That should rectify hopefully